### PR TITLE
Support Ctrl/Alt-Key combos as start key

### PR DESCRIPTION
**Pull request by fingolfin**
This addresses issue #24 *partially*. One can not handle shift, nor are arbitrary combos possible. E.g. `Ctrl-,` is not supported.

### CODE REVIEW

**Comment by MasonProtter** [comment]
This is great, thanks. I'll add some documentation to it and it should be good to go!

File: README.md [hunk 1]
```diff
@@ -226,6 +226,18 @@ Tada!
 </p>
 </details>
 
+# Modifier keys
+<details>
+ <summaryClick me! ></summary>
```
Comment (fingolfin) [inline README.md:231]: Typo in the HTML here

Comment (MasonProtter) [reply README.md:231]: Good catch!

Comment (MasonProtter) [reply README.md:231]: Fixed now

**Pull request merged**